ai_edge_torch/convert/conversion.py

```diff
@@ -88,16 +88,14 @@ def convert_signatures(
   _warn_training_modules(signatures)
 
   exported_programs: torch.export.ExportedProgram = [
-      torch.export.export(
-          sig.module, sig.sample_args, dynamic_shapes=sig.dynamic_shapes
-      )
+      torch.export.export(sig.module, sig.flat_args, dynamic_shapes=sig.dynamic_shapes)
```

Comment:
why aren't we passing kwargs to `torch.export.export` as kwargs?

Reply:
torch_xla does not support exported_program with kwargs to stablehlo, so I need to flatten all the inputs to a tuple before the very first torch.export.export.